src/core/changelogcontents.h

```diff
@@ -0,0 +1,41 @@
+/***************************************************************************
+  changelogcontents.h - Changelog
+
+ ---------------------
+ begin                : Nov 2020
+ copyright            : (C) 2020 by Ivan Ivanov
+ email                : ivan@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef CHANGELOGCONTENTS_H
+#define CHANGELOGCONTENTS_H
+
+#include <QObject>
+
+class ChangelogContents : public QObject
+{
+    Q_OBJECT
+
+  public:
+    explicit ChangelogContents( QObject *parent = nullptr );
+
+    Q_INVOKABLE void request();
+
+    Q_INVOKABLE QString markdown();
```

Comment:
Should we make this a readonly property?

Reply:
Hm, I did it the QML way, but I always get impressed it actually results into more code :)